vocab/quantitykinds/VOCAB_QUDT-QUANTITY-KINDS-ALL-v2.1.ttl

```diff
@@ -1673,6 +1701,20 @@ quantitykind:BodyMassIndex
   rdfs:label "Body Mass Index"@en ;
   skos:altLabel "BMI" ;
 .
+quantitykind:BoilingPoint
```

Comment:
@steveraysteveray Yes, I think I understand the distinction. The manner in which we want to use Boiling Point is to describe reference values about the boiling point of substances (e.g., the boiling point of water) instead of specific measurements. Is this use in the spirit of QUDT's "quantity kind"?

Reply:
Ah, you might also consider the third class of objects, namely qudt:PhysicalConstant. Are you talking about reference, fixed values, such as the speed of light in vacuum? i.e. is your the boiling point of water at atmospheric pressure? 

So your choices are:
1. QuantityKind "BoilingPoint" - just the notion of a boiling point, without a value
2. Quantity "BoilingPoint" for you to use for your measurements, in your defined context
3. PhysicalConstant "BoilingPoint_H20" specified at defined conditions that are well understood by others

Reply:
@steveraysteveray Thanks. We want 1. - Just the notion of a boiling point.